protocol.cpp

```diff
@@ -240,11 +241,16 @@ int redis_protocol::write_command_set(const char *key, int key_len, const char *
             "\r\n"
             "$%u\r\n"
             "%s\r\n"
-            "$%u\r\n", (unsigned int) strlen(expiry_str), expiry_str, value_len);
+            "$%u\r\n", (unsigned int) strlen(expiry_str), expiry_str, total_buffers_len);
+    }
+
+    for (val_list_it it = values_list->begin(); it != values_list->end(); ++it)
+    {
```

Comment:
what is the convention for one-liner for loop?
anyway i think they use to put the `{` in the same line of the `for` statement.